### PR TITLE
Add explanations about new REPL commands to feature list of 2.11

### PR DESCRIPTION
**Pull request by kiritsuku**
I used normal markdown syntax. I hope it is rendered correctly. Didn't know how to render it by myself, thus couldn't test how it looks on docs.scala-lang.org. Is there an easy way to render it by myself?

Review by @som-snytt

### CODE REVIEW

File: 2.11/index.markdown [hunk 1]
```diff
@@ -38,7 +38,237 @@ Improve performance of reflection SI-6638
...
+case class Foo(bar: Int)
+
+// Exiting paste mode, now interpreting.
+
+
+scala> abc.Foo(5)
+res5: abc.Foo = Foo(5)
+```
+
+Furthermore, there is a bug fixed that lead to the problem that no error message is reported when the read code was incomplete ([#2672](https://github.com/scala/scala/pull/2672)).
```
Comment (som-snytt) [inline 2.11/index.markdown]: Lead is the metal, led is the past tense of lead.

Comment (kiritsuku) [inline 2.11/index.markdown]: fixed

**Comment by adriaanm** [comment]
@sschaef, thanks! The [GitHub preview](https://github.com/sschaef/scala/blob/60835e6e217101abb8f7dad11e77b81d3e80787b/2.11/index.markdown) looks fine.

To run jekyll locally:

```
sudo gem install jekyll
jekyll serve & # this takes forever
open http://localhost:4000/scala/2.11/
```

**Comment by som-snytt** [comment]
++1.  Or, as we like to say in Scala, 1 = 2.

I'll try to help when I've learned the process.  And when I've read your doc a couple of times, because frankly, I always forget how the commands work.

One useful thing is javap -raw, which turns off the REPL symbol stripping or flaying.  Sometimes you need the long weird name, or sometimes you can't tell what you're looking at without it.

**Comment by kiritsuku** [comment]
jekyll didn't fully like my code (and me neither! Don't want to know the time I needed to get it to work...), there are small stylistic errors. I'll provide an update soon.

**Comment by som-snytt** [comment]
Thx for taking the bullet.

BTW, keep in mind that sometimes jekyll is Mr Hyde.

**Comment by kiritsuku** [comment]
> BTW, keep in mind that sometimes jekyll is Mr Hyde.

Oh yes, how could I forgot this? (better to say I had no idea.)

**Comment by som-snytt** [comment]
I'll park this info here:  I just realized that the big story was javap support on Java 7 from REPL.  You'd get the "not yet supported." Which is what you still get on 2.10 because standing between it and the backport was a paulp refactor (and I'm lazy).

https://github.com/scala/scala/pull/1637

**Pull request merged**